core/js/typespecs/common_types.ts

```diff
@@ -61,6 +61,7 @@ export const objectTypes = z.enum([
   "organization",
   "api_key",
   "ai_secret",
+  "env_var",
```

Comment:
Let's also add it to `getObjectArticle` so the docs look correct. I wonder if there's a javascript package for this...

Reply:
done